app/helpers/compute_resources_vms_helper.rb

```diff
@@ -142,8 +142,12 @@ def show_vm_name?
     controller_name != 'hosts' && controller_name != 'compute_attributes'
   end
 
-  def new_host?(host)
-    host.try(:new_record?)
+  def new_host?
+    if @host.nil?
```

Comment:
Why not use f.object?  I was hoping we could get rid of `@host`, not increasingly rely on it.